src/libsystemd/libsystemd.sym

```diff
@@ -259,6 +260,7 @@ global:
         sd_bus_message_new_method_errnof;
         sd_bus_message_ref;
         sd_bus_message_unref;
+        sd_bus_message_seal;
```

Comment:
you need to add these two new functions to a new symbol version block at the end of the .sym file, i.e. a new LIBSYSTEMD_235 block chained up with LIBSYSTEMD_234. The way symbol versioning works is that existing blocks are never modified again